src/audio/dai-zephyr.c

```diff
@@ -1605,7 +1597,7 @@ int dai_common_copy(struct dai_data *dd, struct comp_dev *dev, pcm_converter_fun
 	 * in order to avoid high load spike
 	 * if FAST_MODE is enabled, then one period limitation is omitted
 	 */
-	if (!(dd->ipc_config.feature_mask & BIT(IPC4_COPIER_FAST_MODE)))
+	if (dd->non_fast_mode)
```

Comment:
This looks a bit surprising if this helps in performance. You need a mem dereference and check for a bit. Not sure if the modified version is faster, but I guess you've checked the generated assembly?

Reply:
it definitely can speed up, new implementation is 2x faster than previous, and this is an un-changed value during running, no need calculate it per 1ms.

Reply:
@btian1 But IPC4_COPIER_FAST_MODE is a constant so I'd expect the compiler generate pretty efficient code also for original code. In both variants we need to dereference 'dd' and check one bit.

But if you've checked the compiler output, good for me.